restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/DialConferenceTerminationTest.java

```diff
@@ -0,0 +1,350 @@
+    private static final byte[] bytes = new byte[]{118, 61, 48, 13, 10, 111, 61, 117, 115, 101, 114, 49, 32, 53, 51, 54, 53,
+            53, 55, 54, 53, 32, 50, 51, 53, 51, 54, 56, 55, 54, 51, 55, 32, 73, 78, 32, 73, 80, 52, 32, 49, 50, 55, 46, 48, 46,
+            48, 46, 49, 13, 10, 115, 61, 45, 13, 10, 99, 61, 73, 78, 32, 73, 80, 52, 32, 49, 50, 55, 46, 48, 46, 48, 46, 49,
+            13, 10, 116, 61, 48, 32, 48, 13, 10, 109, 61, 97, 117, 100, 105, 111, 32, 54, 48, 48, 48, 32, 82, 84, 80, 47, 65,
+            86, 80, 32, 48, 13, 10, 97, 61, 114, 116, 112, 109, 97, 112, 58, 48, 32, 80, 67, 77, 85, 47, 56, 48, 48, 48, 13, 10};
+    private static final String body = new String(bytes);
+
+    @ArquillianResource
+    URL deploymentUrl;
+
+    private String adminAccountSid = "ACae6e420f425248d6a26948c17a9e2acf";
+    private String adminAuthToken = "77f8c12cc7b8f8423e5c38b035249166";
+
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule(8090); // No-args constructor defaults to port 8080
+
+    private static SipStackTool tool1;
+    private static SipStackTool tool3;
+
+    // Bob is a simple SIP Client. Will not register with Restcomm
+    private SipStack bobSipStack;
+    private SipPhone bobPhone;
+    private String bobContact = "sip:bob@127.0.0.1:5090";
+
+    // George is a simple SIP Client. Will not register with Restcomm
+    private SipStack georgeSipStack;
+    private SipPhone georgePhone;
+    private String georgeContact = "sip:+131313@127.0.0.1:5070";
+
+    private String dialRestcomm = "sip:1111@127.0.0.1:5080";
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        tool1 = new SipStackTool("DialConferenceTool1");
+        tool3 = new SipStackTool("DialConferenceTool3");
+    }
+
+    @Before
+    public void before() throws Exception {
+        bobSipStack = tool1.initializeSipStack(SipStack.PROTOCOL_UDP, "127.0.0.1", "5090", "127.0.0.1:5080");
+        bobPhone = bobSipStack.createSipPhone("127.0.0.1", SipStack.PROTOCOL_UDP, 5080, bobContact);
+
+        georgeSipStack = tool3.initializeSipStack(SipStack.PROTOCOL_UDP, "127.0.0.1", "5070", "127.0.0.1:5080");
+        georgePhone = georgeSipStack.createSipPhone("127.0.0.1", SipStack.PROTOCOL_UDP, 5080, georgeContact);
+    }
+
+    @After
+    public void after() throws Exception {
+        if (bobPhone != null) {
+            bobPhone.dispose();
+        }
+        if (bobSipStack != null) {
+            bobSipStack.dispose();
+        }
+
+        if (georgePhone != null) {
+            georgePhone.dispose();
+        }
+        if (georgeSipStack != null) {
+            georgeSipStack.dispose();
+        }
+        Thread.sleep(3000);
+        wireMockRule.resetRequests();
+        Thread.sleep(4000);
+    }
+
+    private int getActiveConferencesSize() {
+        JsonObject conferences = RestcommConferenceTool.getInstance().getConferences(deploymentUrl.toString(),adminAccountSid, adminAuthToken);
```

Comment:
this may reuse my new countByFilter metthod :)

Reply:
@jaimecasero can you explain how can i use a dao method at this level, is there an api available on top of that ?